makeflow/src/dag_file.c

```diff
@@ -37,4 +61,24 @@ int dag_file_is_sink( const struct dag_file *f )
 		return 1;
 }
 
+int dag_file_exists( const struct dag_file *f )
```

Comment:
Yes, a file is either an input which is checked for existence when we start, or is logged as existing from a check after the node has completed. There is no actual checking within the function. Should we rename the function?